db/migrate/20170621180611_add_population_to_budget_headings.rb

```diff
@@ -0,0 +1,5 @@
+class AddPopulationToBudgetHeadings < ActiveRecord::Migration
+  def change
+    add_column :budget_headings, :population, :integer, default: 0
```

Comment:
Problem is people may not understand on the admin panel difference between empty field and 0. So if they input a 0 assuming that's going to mean "No population" the rule may not work correctly, right?